lib/requester/requester-pool.js

```diff
@@ -14,7 +14,8 @@ RequesterPool = function (options) {
         strictSSL: _.get(options, 'requester.strictSSL'),
         sendBodyWithGetRequests: _.get(options, 'requester.sendBodyWithGetRequests', false),
         followRedirects: _.get(options, 'requester.followRedirects', true),
-        restrictedAddresses: _.get(options, 'restrictedAddresses')
+        restrictedAddresses: _.get(options, 'restrictedAddresses'),
```

Comment:
Now we have two of these IP like confit sent to requester and is very specific network stuff. They might need to be placed somewhere or inside its own namespace in options.

Reply:
This can be done and makes sense, but doing it should be a breaking change and we've just released `7.0`. 

Other option would be to support both `options.restrictedAddresses` and `options.network.{restrictedAddresses,hostIpMap}`.